cfpb/cfgov-refresh
# Adding Redirect for Leadership calendar

Adding Redirect for Leadership calendar


## Changes

- Modified `cfgov/cfgov/urls.py` to adding redirect to Wagtail page.
- Modified `cfgov/v1/util/util.py` to add secondary navigation items.

## Testing

- Download the build dump
- Change the slug of the `CFGov -> About-Us -> Leadership Calendar` page to `leadership-calendar`.
- Visit `http://localhost:8000/about-us/the-bureau/leadership-calendar/` and `http://localhost:8000/about-us/leadership-calendar/`. Both URLs should render the Wagtail Leadership page.

## Screenshots


## Notes

-

## Todos

-

## Checklist
